src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -185,12 +198,15 @@ private String executeCommand(String commandText) throws CommandException, Parse
             throw e;
         }
 
-        if (commandResult == Messages.MESSAGE_SHOWING_HELP) {
+        if (commandResult.equals(Messages.MESSAGE_SHOWING_HELP)) {
             handleHelp();
         }
-        if (commandResult == Messages.MESSAGE_EXITING) {
+        if (commandResult.equals(Messages.MESSAGE_EXITING)) {
             handleExit();
         }
```

Comment:
The use of `==` is intentional. Refer to https://github.com/AY2324S2-CS2103T-W12-3/tp/pull/93/files#r1531535473. Would be good to add a comment in the code to explain why.

Reply:
Reverted to using `==` and comment added to explain